deploy/examples/csi/rbd/storageclass-radosnamespace.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph # namespace:cluster
+spec:
+  failureDomain: osd
+  replicated:
+    size: 3
+    # Disallow setting pool with replica 1, this could lead to data loss without recovery.
```

Comment:
This comment is not accurate since we changed back to a replica 3 pool. For simplicity of the example, how about just remove the comments from the CephBlockPool? They can see comments in other pool examples if needed.

Reply:
done.